### ISSUE TITLE
Dependancy conflict

### ISSUE THREAD

**Issue by salarmehr** [wasinger/htmlpagedom#10]
Thanks for this package. I am trying to use that in one of my Laravel project. I can not install due to dependency hell. Do you plan to use new version of symfony/css-selector (v3.*)?

``````
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install wa72/htmlpagedom v1.2.1
    - Conclusion: remove symfony/css-selector v3.0.3
    - Installation request for wa72/htmlpagedom ^1.2 -> satisfiable by wa72/htmlpagedom[v1.2.0, v1.2.1].
    - Conclusion: don't install symfony/css-selector v3.0.3
    - wa72/htmlpagedom v1.2.0 requires symfony/css-selector >=2.6,<2.8 -> satisfiable by symfony/css-selector[v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, v2.6
.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9].
    - Can only install one of: symfony/css-selector[v2.6.0, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.1, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.10, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.11, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.12, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.13, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.2, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.3, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.4, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.5, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.6, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.7, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.8, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.6.9, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.0, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.1, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.10, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.2, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.3, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.4, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.5, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.6, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.7, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.8, v3.0.3].
    - Can only install one of: symfony/css-selector[v2.7.9, v3.0.3].
    - Installation request for symfony/css-selector == 3.0.3.0 -> satisfiable by symfony/css-selector[v3.0.3].

**Comment by wasinger** [comment]
htmlpagedom is currently not compatible with Symfony 2.8 and greater. See #8

**Comment by wasinger** [comment]
Just released version 1.2.2 that allows 3.x version of symfony/css-selector, so your dependency problem with css-selector should be gone. But note that you still can't use the 3.x version of symfony/dom-crawler, you will need to stick to the 2.8 version of it.

**Comment by salarmehr** [comment]
Thanks a lot for the commits. I could install the package.
